### ISSUE TITLE
Couldn't load project data file on OpenShot

### ISSUE THREAD

**Issue by Shirglt** [OpenShot/openshot-qt#4505]
Hello! I believe this issue was reported earlier by someone else but I'm currently having the same issue which I cannot resolve. 
I am using OpenShot Version 2.6.1 and was saving my project every 2min as the software kept crashing. However, the last time that I saved, the software hanged and said 'Not Responding'. Afterwards, the system crashed again and when I tried to open my project file, it said 'NoneType' Object is not subscriptable. I tried to reinstall OpenShot but the project file still couldn't be opened and it stated 'Couldn't load project data file'. Would greatly appreciate your help please!! The video was quite big and I can't seem to retrieve it back :(

1. Go to File
2. Click on Open Project
3. Error Message Appears once project is selected

Expected behavior:
The project file should have open as I had saved it previously.

System Details:
- OpenShot Version 2.6.1
- Operating System / Distro: Windows 10

Log Files:
[openshot-qt.zip](https://github.com/OpenShot/openshot-qt/files/7419782/openshot-qt.zip)
[libopenshot.zip](https://github.com/OpenShot/openshot-qt/files/7419785/libopenshot.zip)

Screenshots:
![Error Message_OpenShot](https://user-images.githubusercontent.com/93211671/138919603-e2d2097c-7c56-4cfd-8967-24320353fca4.PNG)

**Comment by JacksonRG** [comment]
Looking over the logs, looks like your default project file got moved. I'm looking into how that happened, and if we can find it again.

**Comment by JacksonRG** [comment]
@Shirglt when you look in `C:\Users\shirl\.openshot_qt\recovery`
Do any can you open any of the backup files? Possibly it crashed before finishing saving properly.

**Comment by Shirglt** [comment]
Hi Jackson, I tried to open the backup files from the recovery folder as well but I kept receiving the same error message for all of them.

**Comment by Shirglt** [comment]
Hi Jackson, this was what happened when I tried to open any of the backup files. A message will state that the items within the project cannot be found and a new folder will be created in the recovery folder:

![image](https://user-images.githubusercontent.com/93211671/139093533-b494cb73-8a84-46e1-83b3-781e4682f49a.png)
![image](https://user-images.githubusercontent.com/93211671/139093606-948da7e9-5357-4964-8473-d58c6cddeafe.png)

These are the items that I observe when I enter this new folder:
![image](https://user-images.githubusercontent.com/93211671/139093723-cf39f384-6475-4152-a452-40f19cc42e5e.png)

**Comment by JacksonRG** [comment]
@Shirglt ok. Do you recieve `couldn't load project data file` after clicking `ok` about any missing files?

That error message in your last comment means a file was moved from where Openshot expects it to be. Have you moved folders, or changed file names?

**Comment by Shirglt** [comment]
Hi Jackson, no I did not receive 'couldn't load project file' for all the other backup files from the recovery folder, the message I got is only the screenshot that I sent earlier which stated the file cannot be found.

And nope, the folder was always in the original position where I left it and I never changed the file name of the folder as well..  :/

**Comment by JacksonRG** [comment]
Well then the issue may be a relative path. If you copy that backup project file to wherever your original project was, It may be able to open.

Also your log files shows that when Openshot starts, it can't open the default project. How did you go about reinstalling openshot?

**Comment by Shirglt** [comment]
Ok will try copying the backup project files to the main folder later to see if it works.

For the re-installation, I tried this twice.
On 26 Oct, I simply uninstalled OpenShot from my programs, went to the website again to install OpenShot v2.6.1.
On 27 Oct, I had an issue with OpenShot again cos the software couldn't seem to be opened. I tried to uninstall the program using the same method but I received an error message that said I wasn't able to uninstall it (can't recall exactly what the error message was). I simply went to the website again and tried to install the same program 2.6.1 and the installation worked.
